.github/workflows/workflow.yml

```diff
@@ -71,7 +69,7 @@ jobs:
   push_to_dockerhub:
     runs-on: ubuntu-latest
     env:
-      CLICKHOUSE_VERSIONS: "21.8.15.7, 22.3.20.29, 22.8.19.10, 23.3.4.17, 23.4.4.16, 23.5.3.24, latest"
+      CLICKHOUSE_VERSIONS: "21.8.15.7, 22.3.20.29, 22.8.19.10, 23.3.4.17, latest"
```

Comment:
Non-LTS versions get deprecated very fast. Propose to keep only one non-LTS version - `latest`.